lib/components/Input/date-picker/date-picker.scss

```diff
@@ -0,0 +1,51 @@
+.datepicker {
+    &__input {
+        & label {
+            pointer-events: none;
+            cursor: text;
+        }
+    }
+
+    &__wrapper {
+        position: relative;
+        max-width: 360px;
```

Comment:
@jia-deriv can add this 360px in static.scss
`--temp-static-spacing-360: 360px;`

Reply:
done